tests/unit/apps/test_auxiliary.py

```diff
@@ -405,27 +409,28 @@ def test_auxiliary_raise_error_unknown_series(model):
         "to see if you are using the right track."
     )
     machines = {"0": MagicMock(spec_set=Machine)}
+    app = RabbitMQServer(
```

Comment:
same here. Let's update the test description to something like:

"""Test auxiliary application with unknown series."""